pom.xml

```diff
@@ -13,6 +13,11 @@
   <name>Jenkins jQuery plugin</name>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.625</jenkins.version>
+    <java.level>6</java.level>
```

Comment:
That Jenkins version requieres java.level 7. 
Also probably better to use 1.625.3